lib/longrunning.js

```diff
@@ -294,7 +294,8 @@ Operation.prototype.startPolling_ = function() {
       }
 
       if (!result) {
-        if (rawResponse.metadata.value !== previousMetadataBytes) {
+        if (Buffer.compare(
+              rawResponse.metadata.value, previousMetadataBytes) !== 0) {
```

Comment:
Both seem equal but I wanted to be explicit that we were going to be using the buffer comparison after having the `!==` operator cause a bug. 

Actually, I think `rawResponse.metadata.value.equals(previousMetadataBytes)` would be best. I'm going to change it to this.